gulpfile.js

```diff
@@ -55,9 +56,15 @@ gulp.task('tracks', function () {
              .pipe(gulp.dest('app/chrome/tracks'))
 })
 
+gulp.task('zip', ['copy'], function () {
+  return gulp.src('app/chrome/**')
+             .pipe(zip('chrome.zip'))
+             .pipe(gulp.dest('app'))
+})
+
 gulp.task('copy', ['resources', 'tracks'])
 // build everything once, probably for production
-gulp.task('once', ['browser', 'node', 'copy'])
+gulp.task('once', ['browser', 'node', 'copy', 'zip'])
```

Comment:
@dfarache from the gulp readme:
> Note: The tasks will run in parallel (all at once), so don't assume that the tasks will start/finish in order.

If you want to make sure that you are zipping *after* you are compiling then make the zip function a callback at the end of the 'once' task